alembic/env.py

```diff
@@ -0,0 +1,106 @@
+import logging
+import os
+from logging.config import fileConfig
+
+import yaml
+from alembic import context
+from sqlalchemy import engine_from_config, pool
+
+from felis.datamodel import Schema
+from felis.metadata import MetaDataBuilder
+
+# this is the Alembic Config object, which provides
+# access to the values within the .ini file in use.
+config = context.config
+
+# Interpret the config file for Python logging.
+# This line sets up loggers basically.
+if config.config_file_name is not None:
+    fileConfig(config.config_file_name)
+
+logger = logging.getLogger("alembic.env")
+
+# Set the consdb connection string in the config, overriding the dummy value
+consdb_url = os.getenv("CONSDB_URL")
+if consdb_url is None:
+    raise ValueError("CONSDB_URL not found in environment")
+config.set_main_option('sqlalchemy.url', consdb_url)
+logger.info(f"Using connection string: {consdb_url}")
+
+# Load the Felis schema from the path specified in the alembic.ini file
+sdm_schemas_dir = os.getenv("SDM_SCHEMAS_DIR")
+if sdm_schemas_dir is None:
+    raise ValueError("SDM_SCHEMAS_DIR not found in environment")
+schema_path = f"{sdm_schemas_dir}/yml/cdb_latiss.yaml"
```

Comment:
We'll eventually need to have one of these per instrument.

How do we make sure the `alembic_version` table goes in the schema, not outside it?  Right now, it seems to be going in the default `public` schema.

Reply:
Also, it looks like alembic is taking responsibility for every schema in the database, which would cause it to drop all the Butler tables.  Not so good...

Reply:
> We'll eventually need to have one of these per instrument.

If they all need separate migration histories then we will likely have to give them their own environments with different `env.py` and `alembic.ini` files.

Reply:
> it looks like alembic is taking responsibility for every schema in the database

We can have it ignore certain schemas if their names are known:

https://stackoverflow.com/questions/35342367/how-can-i-ignore-certain-schemas-with-alembic-autogenerate

Reply:
> How do we make sure the `alembic_version` table goes in the schema, not outside it? Right now, it seems to be going in the default `public` schema.

This is easily fixed in the configuration:

```
version_table_schema='your_custom_schema'
```

Though you might actually want that in a separate schema than the one being managed because the migrations themselves may drop/create it.